src/backend/joanie/core/api.py

```diff
@@ -196,16 +196,18 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data, status=201)
 
     @action(detail=True, methods=["POST"])
-    def abort(self, request, uid=None):  # pylint: disable=no-self-use
+    def abort(
+        self, request, id=None
```

Comment:
Did you try with `pk` instead of `id` ? to prevent to disable pylint rules `invalid-name` and `redefined-builtin`.